ppafm/ocl/AFMulator.py

```diff
@@ -269,9 +274,10 @@ def setRho(self, rho=None, sigma=0.71, B_pauli=1.0):
                 print("AFMulator.setRho: Preparing buffers")
             if not np.allclose(B_pauli, 1.0):
                 rho_power = self.rho.power_positive(p=self.B_pauli, in_place=False)
-                self.rho.release()  # Let's not keep the original array in device memory to minimize memory foot print
+                if self.minimize_memory:
```

Comment:
I am afraid that this way, one loses the original FF, when changes the cutoff parameters.

Reply:
The array is not lost. It still exists in the CPU memory. It's just deleted from the GPU memory here when the memory minimization is on.